Dockerfile.rpm.ppc64le

```diff
@@ -0,0 +1,49 @@
+FROM ppc64le/centos:7
+
+RUN yum install -y \
+        vim \
+        rpm-build && \
+    rm -rf /var/cache/yum/*
+
+RUN sed -i 's/include_release_info = 1/include_release_info = 0/' /usr/share/vim/vim74/ftplugin/spec.vim && \
+    echo 'let g:spec_chglog_format = "%a %b %d %Y ".$VENDOR." <".$EMAIL."> ".$VERSION."-".$REVISION' >> /etc/vimrc && \
+    echo 'autocmd VimEnter *.spec execute "normal \\c"' >> /etc/vimrc
+
+ARG USER_ID
+ARG CR_NAME
+ARG CR_EMAIL
+ARG PKG_NAME
+ARG PKG_VERS
+ARG PKG_REV
+ARG PKG_ARCH
+
+VOLUME /dist
+VOLUME /build
+WORKDIR /tmp/$PKG_NAME-$PKG_VERS
+
+ENV VENDOR $CR_NAME
+ENV EMAIL $CR_EMAIL
+ENV NAME $PKG_NAME
+ENV VERSION $PKG_VERS
+ENV REVISION $PKG_REV
+ENV ARCHITECTURE $PKG_ARCH
+
+RUN useradd --non-unique --uid $USER_ID nvidia && chown nvidia: .
+USER nvidia
+
+CMD echo $ARCHITECTURE
```

Comment:
Remove this line :)

Reply:
good eye @flx42! removed.

Reply:
Please, I just did a `diff` with the x86_64 file :)